sunpy/map/sources/adapt.py

```diff
@@ -0,0 +1,37 @@
+"""
+ADAPT Map subclass definitions
+"""
+
+from astropy.time import Time
+
+from sunpy.map.mapbase import GenericMap, SpatialPair
+
+__all__ = ['ADAPTMap']
+
+class ADAPTMap(GenericMap):
+    """
+    ADAPT (Air Force Data Assimilative Photospheric Flux Transport) flux transport model evolves an ensemble of realizations.
```

Comment:
```suggestion
    ADAPT magnetic flux map
     
    The ADAPT (Air Force Data Assimilative Photospheric Flux Transport) flux transport model evolves an ensemble of realizations, using flux transport processes during periods for which there are no observations, and updates the ensemble using the ensemble least-squares (EnLS) data assimilation method to account for model and observational uncertainties. These data products are Carrington maps with an additional third dimension corresponding to the ensemble of models used to produce the ADAPT map.
```

Reply:
I lifted most of this text from the NSO ADAPT webpage. It would be useful if someone who knows more about these data products could look over these as well.